pkgs/by-name/bi/bio-gappa/package.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, autoconf
+, pkg-config
+, libz
+, bzip2
+, xz
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gappa-gen";
+  version = "0.8.4";
+
+  src = fetchFromGitHub {
+    owner = "lczech";
+    repo = "gappa";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-VAp9e5Cu3tTeWSOTZ8kPG1eOyocYZiJsy/s+MWbCMLI=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    autoconf
+  ];
+
+  buildInputs = [
+    libz
+    bzip2
+    xz
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ../bin/gappa $out/bin
```

```suggestion
    install -Dm755 ../bin/gappa $out/bin/gappa
```

Then you can remove `mkdir`. `install` is prefered over `mkdir` `mv` or `cp`.